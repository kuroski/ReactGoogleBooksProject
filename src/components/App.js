import React, { Component } from 'react';

class App extends Component {
  render() {
    return (
      <div className="App">
        Ola mundo
      </div>
    );
  }
}

export default App;
