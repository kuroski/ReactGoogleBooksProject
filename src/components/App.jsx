import React, {Component} from 'react'

class App extends Component {
  componentWillMount() {
    if(!window.localStorage.getItem('favoritedBooks'))
      window.localStorage.setItem('favoritedBooks', JSON.stringify([]))
  }

  render() {
    return (
      <div className="App">
        {this.props.children}
      </div>
    )
  }
}

export default App
