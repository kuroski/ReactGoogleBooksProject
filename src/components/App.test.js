import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const AppTree = renderer.create(
    <App />
  ).toJSON();
  expect(AppTree).toMatchSnapshot();
});
