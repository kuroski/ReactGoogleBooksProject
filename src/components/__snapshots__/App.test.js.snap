exports[`test renders correctly 1`] = `
<div
  className="App">
  Ola mundo
</div>
`;
